docs/source/index.rst

```diff
@@ -35,22 +35,30 @@ Quickstart
 2. Find label errors in your data
 ---------------------------------
 
-cleanlab's :py:meth:`find_label_issues <cleanlab.filter.find_label_issues>` function tells you which examples in your dataset are likely mislabeled. At a minimum, it expects two inputs --- your data's given labels, `labels`, and predicted probabilities, `pred_probs`, from some trained classification model. These must be out-of-sample predictions where the data points were held out from the model during training, which can be :ref:`obtained via cross-validation <pred_probs_cross_val>`.
+cleanlab finds issues in *any dataset that a classifier can be trained on*. The cleanlab package *works with any model* by using model outputs (predicted probabilities) as input -- it doesn't depend on which model created those outputs.
 
-Setting `return_indices_ranked_by` in this function instructs cleanlab to return the indices of potential mislabeled examples, ordered by the likelihood of their given label being incorrect. This is estimated via a *label quality score*, which for example can be specified as ``'self_confidence'`` (predicted probability the given label).
+If you're using a scikit-learn-compatible model (option 1), you don't need to train a model -- you can pass the model, data, and labels into :py:meth:`CleanLearning.find_label_issues <cleanlab.classification.CleanLearning.find_label_issues>` and cleanlab will handle model training for you. If you want to use any non-sklearn-compatible model, e.g. from huggingface, keras, pytorch, etc. (option 2), you can input the trained model's out-of-sample predicted probabilities into :py:meth:`find_label_issues <cleanlab.filter.find_label_issues>`. Examples for both options are below.
```

Comment:
```suggestion
If you're using a scikit-learn-compatible model (option 1), you don't need to train a model -- you can pass the model, data, and labels into :py:meth:`CleanLearning.find_label_issues <cleanlab.classification.CleanLearning.find_label_issues>` and cleanlab will handle model training for you. If you want to use any non-sklearn-compatible model (option 2), you can input the trained model's out-of-sample predicted probabilities into :py:meth:`find_label_issues <cleanlab.filter.find_label_issues>`. Examples for both options are below.
```

Reply:
we do have sklearn-compatible examples of: huggingface, keras, pytorch

Reply:
reasoning? examples are helpful since lots of people use those package and may be curious if they will work with cleanlab. or do you see an error?

Reply:
because it makes it sound to me like these cannot be used with cleanlearning when we have specific examples that show how to use each of these with cleanlearning